pandas/io/formats/style.py

```diff
@@ -1459,7 +1487,7 @@ def pipe(self, func, *args, **kwargs):
         return com.pipe(self, func, *args, **kwargs)
 
 
-def _is_visible(idx_row, idx_col, lengths):
+def _is_visible(idx_row, idx_col, lengths) -> bool:
```

Comment:
can the inputs by typed?

Reply:
The problem is that ```mypy``` is giving false positives, I typed the 'row' as ```str``` and the 'col' as ```int``` and it went through with no errors.